### DISCUSSION TITLE
OMR build for BananaPi BPI-R4 ?

### DISCUSSION THREAD

**Discussion by packlet** [Ysurac/openmptcprouter#3677] [General]
i see some development occuring for this, but can't find where to download any complied FW for the BananaPi BPI-R4.  What's the status on this and if it's available, where can i find it?  Thanx.

**Comment by Ysurac** [comment]
It's in snapshot: https://snapshots.openmptcprouter.com/6.6/
But doesn't seems to work for now.

**Reply by GregSumner** [reply]
Hi, I have just successfully booted OMR on my BPI-R4 from SD and nand. I built from develop@4f110624e with 6.6 kernel after copying across target files from 24.10 rc5. Attached is a bootlog of a snapshot failing and a successful boot. Just guessing from the config diff, maybe CONFIG_UIMAGE_FIT_BLK was needed?

The interface came up on eth1 on the SFP port marked SFP LAN. Install from the bootloader to the nand worked well too.

Please let me know what you'd like from me to get support in. All I did was update the mediatek target folder to 24.10 rc5. diff attached.

[6.6-r0+28308-c06d4df974-bootlog.txt](https://github.com/user-attachments/files/18468613/6.6-r0%2B28308-c06d4df974-bootlog.txt)
[develop-4f110624ed-bootlog.txt](https://github.com/user-attachments/files/18468614/develop-4f110624ed-bootlog.txt)
[diff.txt](https://github.com/user-attachments/files/18468647/diff.txt)
